src/app/testQueryFunctionFactory.ts

```diff
@@ -0,0 +1,23 @@
+import DatabaseService from '../interfaces/DatabaseService';
+import ResultsEvent from '../interfaces/ResultsEvent';
+import { getTestResultsByVrm, getTestResultsByVin, getTestResultsByTestId } from './databaseService';
+import TestRecord from '../interfaces/queryResults/test/testRecord';
+
+export default (
```

Comment:
Ok, I can see that we entirely mocked the DatabaseService which is the contract we are testing so perhaps testing the queries are not needed

Reply:
I think its fine now its done.

Reply:
Thanks for clarifying 👍